src/interactive/accordion.story.jsx

```diff
@@ -5,328 +5,496 @@ import { storiesOf } from '@storybook/react';
+										the undoubtable source.
+									</p>
 								</div>
-							} />,
+							}
+						/>,
 						<AccordionPanel
-							label='Second with left icon'
+							label="Second with left icon"
 							panelContent={
 								<div>
-									<div className='runningText'>
+									<div className="runningText">
 										<p>Any kind of content can go in here, even inputs.</p>
 									</div>
-									<div className='chunk'>
-										<label htmlFor='test-textinput'>I'm a label</label>
-										<input id='test-textinput' type='text' placeholder='Input placeholder' />
+									<div className="chunk">
+										<label htmlFor="test-textinput">I'm a label</label>
+										<input
+											id="test-textinput"
+											type="text"
+											placeholder="Input placeholder"
+										/>
 									</div>
 								</div>
-							} />,
+							}
+						/>,
 						<AccordionPanel
-							label='Third with left icon'
+							label="Third with left icon"
 							panelContent={
-								<div className='runningText'>
-									<p>Classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source.</p>
+								<div className="runningText">
+									<p>
+										Classical Latin literature from 45 BC, making it over 2000
+										years old. Richard McClintock, a Latin professor at
+										Hampden-Sydney College in Virginia, looked up one of the
+										more obscure Latin words, consectetur, from a Lorem Ipsum
+										passage, and going through the cites of the word in
+										classical literature, discovered the undoubtable source.
+									</p>
 								</div>
-							} />,
-					]}/>
+							}
+						/>,
+					]}
+				/>
 			</div>
 		)
 	)
 	.addWithInfo(
 		'Panel open by default',
 		'Sets first panel to be open by default via the AccordionPanel props',
 		() => (
-			<div className='span--100 padding--all'>
+			<div className="span--100 padding--all">
 				<AccordionPanelGroup
 					accordionPanels={[
 						<AccordionPanel
 							isOpen
-							label='First (open by default)'
+							label="First (open by default)"
 							panelContent={
-								<div className='runningText'>
-									<p>Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source.</p>
+								<div className="runningText">
+									<p>
+										Contrary to popular belief, Lorem Ipsum is not simply random
```

Comment:
another nitpick - could we store this text content in a variable above the tests to make the test code easier to read/diff? This can be a follow up ticket.